x-pack/legacy/plugins/index_lifecycle_management/__jest__/components/edit_policy.test.js

```diff
@@ -383,14 +374,14 @@ describe('edit policy', () => {
     });
   });
   describe('cold phase', () => {
-    test('should show positive number required error when trying to save cold phase with 0 for after', () => {
```

Comment:
I think this is still a valid test as well. However, it would need to be updated to something like `'should show positive number required error when trying to save cold phase with negative number'`, and line 391 would need to be updated with a negative number.

Reply:
For all three `cold, warm, and delete` tests that checks for a positive number with 0 for after I've changed the test to       `expectedErrorMessages(rendered, [])` . I deleted the variable `positiveNumbersEqualAboveZeroErrorMessage` because now 0 is a valid input. Therefore, there wouldn't be any errors. When the jest ran it expected an error with `expectedErrorMessages(rendered, [positiveNumbersEqualAboveZeroErrorMessage])`

Reply:
I understand. I was suggesting that testing for negative numbers is still valid.

Reply:
Update: You can ignore this. I see we are already checking for negative numbers as a test below this:
```
    test('should show positive number required error when trying to save cold phase with -1 for after', () => {
```